### ISSUE TITLE
Add `maincontent` id to Interactive layouts?

### ISSUE THREAD

**Issue by bryophyta** [guardian/dotcom-rendering#5862]
As far as I understand it, our 'skip to main content' button targets an element with `id='maincontent'`. For some layouts, e.g. `InteractiveImmersiveLayout` and `FullPageInteractiveLayout`, it looks like we are currently relying on an element with that `id` being present somewhere in the embedded interactive content.

Is this how it should be? Or is there an argument for putting `id='maincontent'` in the DCR code to ensure that it is present? (e.g. on the `<main>` element for these layouts)

**Comment by bryophyta** [comment]
just seen that this has already been addressed by @DanielCliftonGuardian 2 days ago! 🥳 (I was looking at an out of date branch.)
